sdk/storage/storage-blob/src/BlobClient.ts

```diff
@@ -562,6 +563,155 @@ export interface DownloadFromBlobOptions {
   parallelism?: number;
 }
 
+export interface BlobSetAccessControlOptions {
+  /**
+   * An implementation of the `AbortSignalLike` interface to signal the request to cancel the operation.
+   * For example, use &commat;azure/abort-controller create an `AbortSignal`.
+   *
+   * @type {AbortSignalLike}
+   * @memberof BlobSetAccessControlOptions
+   */
+  abortSignal?: AbortSignalLike;
```

Comment:
Is this only in track2 that we make abortSignal in Options structure? As I know in track1 it's in per method?

Reply:
Yes, only in track2

Reply:
Yes, only in track2

---
In reply to: [318943959](https://github.com/Azure/azure-sdk-for-js/pull/4874#discussion_r318943959) [](ancestors = 318943959)